src/.vuepress/components/ExtensionList.vue

```diff
@@ -1,26 +1,41 @@
 <template>
-	<div>
+	<div class="extension-list">
 		<div v-for="extensionGroup in extensions" :key="extensionGroup[0].lang">
-			<h3>{{ langName(extensionGroup[0].lang) }}</h3>
+			<h3>
+				{{ langName(extensionGroup[0].lang).replace("English (English)", "English") }}
```

Comment:
You could probably just check if the language code is "en" then just change the function that is being called
```suggestion
				{{ extensionGroup[0].lang === "en" ? simpleLangName(extensionGroup[0].lang) : langName(extensionGroup[0].lang) }}
```